src/main/resources/templates/macros/json/embeddings.vm

```diff
@@ -31,39 +31,63 @@
 
 
 #macro(includeImageEmbedding, $embedding, $image_type, $index)
-  #set($Quote = '"')
-  #includeExpandableEmbedding($embedding, $image_type, "<img src=${Quote}embeddings/$embedding.getFileName()${Quote}>", $index)
+  #set($index = $index + 1)
+  <div class="embedding indention">
+    #set($embeddingId = $counter.next())
```

Comment:
The `includeExpandableEmbedding` macro was used like this:

    #includeExpandableEmbedding($embedding, $format,
                <pre>$embedding.getEscapedDecodedData()</pre>", $index) 

Escaping all html broke the  macro because the value of `$content` would also be escaped. I fixed this by inlining `includeExpandableEmbedding` into every macro that used it. This results in the heavy duplication that you're looking at now.